packages/scss/CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
-# Change log
+# Changelog
 
-## 0.0.0-VERSIONPLACEHOLDER [release](https://github.com/LuccaSA/lucca-front/releases/tag/0.0.0-VERSIONPLACEHOLDER)
-### Bug fixes
\ No newline at end of file
+## v0.1.2
+
+### Breaking changes
+- `input` & `textarea` are now merged in a single component : `textfield`
+- `main` layout structure and modifiers are simplified  
```

Comment:
We should be a bit more specific, precising that margin are now the `.container` responsibility